AccessCheckoutSDK/AccessCheckoutSDK/client/CardValidationConfig.swift

```diff
@@ -11,12 +11,10 @@ public struct CardValidationConfig: ValidationConfig {
     let expiryDate: AccessCheckoutUITextField?
```

Comment:
Can you move the `CardValidationConfig` under client > validation > card to match what was done for `CvcOnlyValidationConfig`